ddtrace/internal/telemetry/writer.py

```diff
@@ -335,7 +329,19 @@ def add_configs_changed(self, cfg_names):
 
     def _telemetry_entry(self, cfg_name: str) -> Tuple[str, str, _ConfigSource]:
         item = config._config[cfg_name]
-        if cfg_name == "_trace_sample_rate":
+        if cfg_name == "_trace_enabled":
```

Comment:
Do we need these `if statements`? Can we just set `name` to `item.name`? Convert `item.value()` to a str(...).lower() if it is a boolean?

Reply:
Yeah, this code could be cleaned up with a dictionary mapping `cfg_name` to `name`.

Reply:
Yeah I had at [one point](https://github.com/DataDog/dd-trace-py/pull/7861/commits/fc754e0d90fc48b95b3e2cd7dc5118f00dc6dd06) added the telemetry name to the ConfigItem class. We could go back to something like that. I think I had moved away from that approach as it felt a bit odd to be upfront tying the telemetry implementation to the configuration item (telemetry reporting name seems more like an implementation detail).

Happy to maintain a mapping somewhere if either of you have a suggestion!